### PR TITLE
Add & use SBB font families

### PR DESCRIPTION
**Pull request by edellucien**

### CODE REVIEW

File: src/WebComponent.scss [hunk 1]
```diff
@@ -1,3 +1,38 @@
...
+    url('utils/fonts/SBBWeb-Thin.woff') format('woff');
+}
+
+@font-face {
+  font-family: 'SBB Web UltraLight';
+  src:
+    url('utils/fonts/SBBWeb-UltraLight.woff2') format('woff2'),
+    url('utils/fonts/SBBWeb-UltraLight.woff') format('woff');
+}
+
```
Comment (oterral) [inline src/WebComponent.scss]: use a function like in moco-frontend :
```
/**
* Load fonts.
*/
@mixin loadFonts($paths...) {
  @each $family in $paths {
    @font-face {
      font-family: #{$family};
      src: url('fonts/#{$family}.eot');
      src:
        url('fonts/#{$family}.woff2') format('woff2'),
        url('fonts/#{$family}.woff') format('woff');
      font-weight: normal;
      font-style: normal;
    }
  }
}

@include loadFonts('SBBWeb-Roman');
@include loadFonts('SBBWeb-Bold');
```

Comment (edellucien) [reply src/WebComponent.scss]: Mr updated as proposed :) Thanks

File: src/components/TrafimageMaps/TrafimageMaps.scss [hunk 2]
```diff
@@ -1,7 +1,7 @@
 @import '../../globals.scss';
 
 .tm-trafimage-maps {
-  font-family: 'SBBWeb Roman', 'Helvetica Neue', Helvetica, Arial, sans-serif;
+  font-family: 'SBBWeb-Roman', 'Helvetica Neue', Helvetica, Arial, sans-serif;
```
Comment (oterral) [inline src/components/TrafimageMaps/TrafimageMaps.scss]: In the ticket it's written ,' Arial as fallback'

Comment (edellucien) [inline src/components/TrafimageMaps/TrafimageMaps.scss]:
```suggestion
  font-family: 'SBBWeb-Roman', Arial, sans-serif;
```

Comment (edellucien) [reply src/components/TrafimageMaps/TrafimageMaps.scss]: I changed with a suggestion! Good catch

**Pull request merged**